data/maps/MtPyre_Summit/scripts.inc

```diff
@@ -622,4 +622,3 @@ MtPyre_Summit_Text_HoennTrioTale:
 	.string "becalmed the two enraged POKéMON.\p"
 	.string "Then, its duty done, the green POKéMON\n"
 	.string "flew off into the wild blue yonder…$"
-
```

Comment:
While this is indeed an extraneous newline, it should be removed in pret, not here

Reply:
Undone locally